pkg/getty/README.md

```diff
@@ -10,7 +10,7 @@ If you want a console getty, add the following to your `moby.yml`:
 ```
 services:
   - name: getty
-    image: "linuxkit/getty:e36b5e794256c2a56c000035f73a4d46f7ffb3e6"
```

Comment:
The `Makefile` has to be in the hash, because it can change the thing that is built (eg different compile options). I do think we should move the READMEs to `docs/` and/or comments in the examples, and definitely remove specific hashes from them.